src/OmniSharp.DotNet/Projects/ProjectSearcher.cs

```diff
@@ -1,66 +1,86 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
-using System.Linq;
 using Microsoft.DotNet.ProjectModel;
 
 namespace OmniSharp.DotNet.Projects
 {
     public class ProjectSearcher
     {
-        public static IEnumerable<string> Search(string solutionRoot)
+        public static IEnumerable<string> Search(string directory)
         {
-            return Search(solutionRoot, maxDepth: 5);
+            return Search(directory, maxDepth: 5);
         }
 
-        public static IEnumerable<string> Search(string solutionRoot, int maxDepth)
+        public static IEnumerable<string> Search(string directory, int maxDepth)
```

Comment:
Note that this function was inconsistent before my changes. Sometimes it would return a directory path to a project.json files, and sometimes it would return multiple _file_ paths to project.json files. Now, it just returns file paths to project.json files.